### PR TITLE
imageId example for Azure Compute ARM

### PR DESCRIPTION
**Pull request by bostko**

### CODE REVIEW

File: guide/locations/_azure-ARM.md [hunk 1]
```diff
@@ -95,6 +95,17 @@ The loginUser can be anything, as long as it's specified.
 The `overrideAuthenticateSudo: true` key tells Apache Brooklyn that default on Azure images do not have passwordless sudo 
 configured by default.
 
+    imageId: westeurope/OpenLogic/CentOS/7.3
+
+Listing images in Azure CLI [https://docs.microsoft.com/en-us/cli/azure/vm/image#list-skus](https://docs.microsoft.com/en-us/cli/azure/vm/image#list-skus).
```
Comment (drigodwin) [inline guide/locations/_azure-ARM.md]: Why should users use this over [cloud:list-images](http://docs.cloudsoftdev.io/operations/tools/index.html#cloud-compute-tools)

File: guide/locations/_azure-ARM.md [hunk 2]
```diff
@@ -95,6 +95,17 @@ The loginUser can be anything, as long as it's specified.
...
+A VM Image can be specified using Brooklyn location config `imageId`.
+For Azure ARM, Apache Brooklyn expects `imageId` in one of the following formats:
+
+- `location/group/storage/offer/name`
+- `location/publisher/offer/sku`
+
+Consider using imageId sku instead of `osVersion` or `osVersionRegex`.
```
Comment (drigodwin) [inline guide/locations/_azure-ARM.md]: Why should they consider this? I would remove this line.

**Comment by tbouron** [comment]
@bostko Can you reply and update your PR, based on @drigodwin's comments?

**Comment by bostko** [comment]
@tbouron I agree we should look why `osVersionRegex: 7\..*` does not choose the latest and probably fire a bug instead of adding such note.
Closing PR. I will create a jira issue at some point.

**Pull request closed without merging**